### DISCUSSION TITLE
Service release 7 for .net 7

### DISCUSSION THREAD

**Discussion by FM1973** [dotnet/maui#16029] [General]
When can we expect the next service release for .net 7?
This is badly needed.
The last version 7.0.86 still has some severe bugs regarding the grid control, besides others bugs.
@mattleibow  @hartez  @samhouts  Can you please let us know?
Do we have to tell our customers to wait for christmas?
Thanks for the info!

**Comment by hartez** [comment]
Current target is next week.

**Reply by FM1973** [reply]
Good to know. Thanks a lot @hartez !

**Reply by JohnHDev** [reply]
Do you have a list of bugs fixed in .NET 7? 
Has this PR been merged back? https://github.com/dotnet/maui/pull/15036
